lib/matplotlib/rcsetup.py

```diff
@@ -818,6 +836,12 @@ def validate_cycler(s):
         norm_prop = _prop_aliases.get(prop, prop)
         cycler_inst.change_key(prop, norm_prop)
 
+    for key, vals in cycler_inst.by_key().items():
+        _prop_validators[key](vals)
+
+    if 'color' not in cycler_inst.keys:
+        cycler_inst = cycler_inst * cycler('color', ['k'])
```

Comment:
This is what is currently in the code.  I suggest we merge this (to un break tests) and then discuss this in a separate issue.